test/metric/metric_value_query.go

```diff
@@ -34,9 +35,14 @@ func GetMetricFetcher(metricName string) (MetricValueFetcher, error) {
 }
 
 type MetricValueFetcher interface {
+	// Fetch returns the values associate with the metric within the past 10 minutes
```

Comment:
I understand return types were not exactly typed.. but I'm mostly on the side that believes comments shouldn't be needed if code is written in a certain way and should be avoided. Would it be helpful if return types were []MetricValue or so instead of this comment? ditto for all other comments added here. Some seems very redundant and/or can be addressed by changing method names

Reply:
I'm in the same boat with ya. However, I also believe we will never achieve "perfect code" that will help people understand at first sight. Therefore, comments. But yep I agree I intend to change it in "refactor" PR. If everyone agree,I can change it now too !

Reply:
If you agree that the comments are redundant, then you should remove them.